web/test/pages/onboarding/onboarding-starting.test.tsx

```diff
@@ -108,7 +108,7 @@ describe("onboarding - starting a business", () => {
         useMockRouter({ isReady: true, query: { page: "2" } });
         const { page } = renderPage({ userData });
 
-        page.chooseEssentialQuestionRadio(industryId, 0);
+        page.chooseEssentialQuestionRadio(industryId, industryId === "employment-agency" ? 1 : 0);
```

Comment:
pebble - why are we changing the logic here? what happens when with the original value. If we need have one off logic, then I think the best think to do here is to separate the tests insteag of using `it.each`